src/modding/ModManager.cs

```diff
@@ -857,6 +1308,251 @@ private void OpenInfoUrlPressed()
+            {
+                if (!string.IsNullOrWhiteSpace(currentIncompatibleMod))
+                {
+                    infoText += "* " + currentIncompatibleMod + "\n";
+                }
+            }
+        }
+        else
+        {
+            infoText += TranslationServer.Translate("NO_MOD_INCOMPATIBLE");
+        }
+
+        otherModInfoDialog.DialogText = infoText;
+        otherModInfoDialog.PopupCenteredShrink();
+    }
+
+    private void OnLoadOrderPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        if (selectedMod is null)
+        {
+            return;
+        }
+
+        otherModInfoDialog.WindowTitle = TranslationServer.Translate("LOAD_ORDER");
+        var infoText = string.Empty;
+
+        var currentModLoadBefore = selectedMod.Info.LoadBefore;
+        var currentModLoadAfter = selectedMod.Info.LoadAfter;
+        if (currentModLoadBefore != null || currentModLoadAfter != null)
+        {
+            if (currentModLoadAfter != null)
+            {
+                infoText += TranslationServer.Translate("MOD_LOAD_AFTER") + "\n";
+                foreach (string currentLoadAfterMod in currentModLoadAfter)
+                {
+                    if (!string.IsNullOrWhiteSpace(currentLoadAfterMod))
+                    {
+                        infoText += "* " + currentLoadAfterMod + "\n";
+                    }
+                }
+            }
+
+            // If there both 'load before' and 'load after' is going to display add a empty line between them
+            if (currentModLoadBefore != null && currentModLoadAfter != null)
+            {
+                infoText += "\n";
+            }
+
+            if (currentModLoadBefore != null)
+            {
+                infoText += TranslationServer.Translate("MOD_LOAD_BEFORE") + "\n";
+                foreach (string currentLoadBeforeMod in currentModLoadBefore)
+                {
+                    if (!string.IsNullOrWhiteSpace(currentLoadBeforeMod))
+                    {
+                        infoText += "* " + currentLoadBeforeMod + "\n";
+                    }
+                }
+            }
+        }
+        else
+        {
+            infoText += TranslationServer.Translate("MOD_NO_LOAD_ORDER");
+        }
+
+        otherModInfoDialog.DialogText = infoText;
+        otherModInfoDialog.PopupCenteredShrink();
+    }
+
+    private void GalleryRightArrowPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        selectedModPreviewImagesContainer.CurrentTab += 1;
+        galleryLabel.Text = (selectedModPreviewImagesContainer.CurrentTab + 1) + "/" +
+            selectedModPreviewImagesContainer.GetTabCount();
+        galleryRightButton.Disabled = selectedModPreviewImagesContainer.CurrentTab >=
+            selectedModPreviewImagesContainer.GetTabCount() - 1;
+        galleryLeftButton.Disabled = selectedModPreviewImagesContainer.CurrentTab <= 0;
```

Comment:
This looks very suspiciously like duplicated code that belongs in a helper method.